taxi_service/settings.py

```diff
@@ -133,3 +133,7 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+AUTH_USER_MODEL = "taxi.Driver"
```

Comment:
The `AUTH_USER_MODEL` setting is defined twice in the file. This redundancy should be removed to avoid potential confusion or errors. Ensure that `AUTH_USER_MODEL = "taxi.Driver"` is only declared once.